app/gui2/src/components/visualizations/ScatterplotVisualization.vue

```diff
@@ -539,16 +672,18 @@ useEvent(document, 'keydown', bindings.handler({ zoomToSelected: () => zoomToSel
 <template>
   <VisualizationContainer :belowToolbar="true">
     <template #toolbar>
+      <SvgButton name="add" title="Enable Selection" @click="enableSelection()" />
```

Comment:
I think it's confusing to use the same icon as the add-node buttons for something unrelated.

Reply:
Yes I agree! I'll talk to Ad about what icon is best!